examples/credentials/sd_jwt_vc_unsecured.json

```diff
@@ -0,0 +1,6 @@
+{
+  "vct": "https://credentials.example.com/identity_credential",
+  "given_name": "John",
+  "family_name": "Doe",
+  "birthdate": "1940-01-01"
+}
```

Comment:
Hmm, true :/ I always like to be sure to add a newline at the end of every line to avoid weird things from happening (some tools break without newline at the end of a line), but let's just keep things how they were in the repo.